.sql/data.sql

```diff
@@ -0,0 +1,246 @@
+-- phpMyAdmin SQL Dump
+-- version 4.6.6deb1+deb.cihar.com~xenial.2
+-- https://www.phpmyadmin.net/
+--
+-- Host: localhost
+-- Czas generowania: 24 Paź 2018, 00:34
+-- Wersja serwera: 5.7.20-0ubuntu0.16.04.1
+-- Wersja PHP: 7.2.11-2+ubuntu16.04.1+deb.sury.org+1
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET time_zone = "+00:00";
+
+--
+-- Baza danych: `adfind`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Struktura tabeli dla tabeli `Category`
+--
+
+CREATE TABLE `Category` (
```

Comment:
Ja bym tutaj scalił tabele Category oraz Subcategory w jedną bo przechowują to samo. Dorzuć kolumnę np parent_category_id która mogłaby być nullem. Jeśli jest nullem to znaczy, że jest to kategoria główna. Tym sposobem upraszczasz sobie strukturę bazy jak i sam kod (masz tylko pojedyncza klasę encji)